messages/eslintrc-incompat.js

```diff
@@ -102,6 +102,15 @@ A config object is using the "reportUnusedDisableDirectives" key, which is not s
 
 Flat config uses "linterOptions.reportUnusedDisableDirectives" to specify files to ignore.
 
+Please see the following page for information on how to convert your config object into the correct format:
+https://eslint.org/docs/latest/use/configure/migration-guide#linter-options
+`,
+
+    reportUnusedInlineConfigs: `
```

Comment:
Similar here. This is a template for when eslintrc options are used in flat config files, so I'm not sure if it makes sense to handle the new option here as it didn't exist in eslintrc.